src/TeaAsAService/Tea.Test/Web/UserControllerIntegrationFixture.cs

```diff
@@ -86,5 +86,29 @@ public async Task UpdateUser_UpdatesWithoutValidModelThrowsError()
 
             Assert.NotNull(response);
         }
+
+        [Fact]
```

Comment:
Its probably worth putting at least one failure test in as well, set a password to be null or something and make sure we fail the postAndAsset